circuit-benchmarks/src/pi_circuit.rs

```diff
@@ -47,17 +43,17 @@ mod tests {
         ));
         let public_inputs = circuit.0.instance();
         let instance: Vec<&[Fr]> = public_inputs.iter().map(|input| &input[..]).collect();
-        let instances = &[&instance[..]][..];
+        let instances = &[&instance[..]];
```

Comment:
Hi, @han0110  , the bug is over here. A very small one.
And I've updated the branch.